### PR TITLE
Add basic setup for the Add entry page

### PR DESCRIPTION
**Pull request by riittagerhards**
Add basic structure for the Add entry page including:
-import of navigation and button components
-set up the form element
-show page in Storybook

part of #9

image

https://dear-diary-r-add-entry-rkn77hq.herokuapp.com/storybook/?path=/story/pages-addentry--default

Open to-do's in #9:
-add cloudinary for image upload
-add functions

### CODE REVIEW

File: src/app/components/UploadImage/UploadImageIcon.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="42" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lajanzen) [inline src/app/components/UploadImage/UploadImageIcon.svg:1]:
```suggestion
<svg width="42" height="40" viewBox="0 0 42 40" fill="none">
```

Comment (lajanzen) [inline src/app/components/UploadImage/UploadImageIcon.svg:1]: Has this file been formatted?

File: src/app/pages/AddEntry/AddEntry.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <div className={styles.container}>
+      <Navigation headerTitle={'How was your day?'} />
+      <form className={styles.form}>
+        <input
+          className={styles.titleInput}
+          type="text"
+          placeholder="... add title ..."
+          required
+        ></input>
```
Comment (lajanzen) [inline src/app/pages/AddEntry/AddEntry.tsx:16]: Do you need this closing tag here?
```suggestion
        / >
```

**Review by lajanzen — COMMENTED** [review]
Nice!

File: src/app/components/UploadImage/UploadImageIcon.svg (hunk 1 above)
Comment (riittagerhards) [reply src/app/components/UploadImage/UploadImageIcon.svg]: Yes!

**Pull request merged**